franklin/builder/tests.py

```diff
@@ -164,16 +164,13 @@ def setUp(self):
 
     def test_tag_build_path(self):
         """ Test that object instantiation saves correct path. """
-        expected = "{base}/{owner}/{site}/{tag}".format(
-            base=os.environ['BASE_PROJECT_PATH'], owner=self.site.owner.name,
-            site=self.site.name, tag='v102')
+        expected = "{site}/{tag}".format(site=self.site.github_id, tag='v102')
         self.assertEqual(self.tag_build.path, expected)
 
     def test_branch_build_path(self):
         """ Test that object instantiation saves correct path. """
-        expected = "{base}/{owner}/{site}/{git_hash}".format(
-            base=os.environ['BASE_PROJECT_PATH'], owner=self.site.owner.name,
-            site=self.site.name, git_hash='asdf1234')
+        expected = "{site}/{git_hash}".format(site=self.site.github_id,
+                                              git_hash='asdf1234')
         self.assertEqual(self.branch_build.path, expected)
```

Comment:
Code-wise this isn't a huge change, but I spent some time confirming that github repo ids seem to NEVER change. You can transfer ownership, rename them, etc. The id will be the same. If you delete a repo, it doesn't appear to be available for anyone else to use. Github doesn't appear to reuse old ids. 

@CorneliusIV - Builder needs to change what it does with `path`. I think @JulianGindi wants an ENV var to set the base folder that everything will go into. You'll then just append the path API sends to the end of that.
https://github.com/istrategylabs/franklin-build/blob/master/api.go#L162